Decision records/05 - We use Details and List to describe index- and show-style route components.md

```diff
@@ -0,0 +1,43 @@
+# We use `Details` and `List` to describe index- and show-style route components
```

Comment:
cc/ @ry5n, I think I heard you were similarly thinking about naming of these pages from the UX side.

Reply:
Thanks for the heads up. I ❤️ this, they’re just more human than “index” and “show”. I’ll add an item to discuss this with the Polaris team and switch to using these terms in docs and UX discussion.

cc @selenehinkley

Reply:
updated this in the web workshop content to reflect the decision 👍 

I would like some suggestion on what to name page that's not necessary a Resource thou?
ie. the app has one base page under `Home` section, I renamed it from `HomeIndex` to `HomeDetails`, but it actually feels wrong since Home is not a resource.

Reply:
I'm just creeping here... :) First, this decision makes a lot of sense. Second, can you _please_ change the title to:

'We use `Details` and `List` to describe and show- and index-style route components'

to maintain respective order of the terms?